server_admin/topics/clients/client-policies.adoc

```diff
@@ -128,32 +124,31 @@ One of several purposes for this executor is to realize security requirements of
-There are two _global profiles_ configured in the {project_name} by default with pre-configured executors compliant with the FAPI Baseline and FAPI Advanced specifications.
-More details in the FAPI section of the link:{adapterguide_link}#_fapi-support[{adapterguide_name}].
+A profile consists of several executors, which can realize a security profile like FAPI. Profile can be configured by the Admin REST API (Admin Console) together with its executors.
+Two _global profiles_ exist and they are configured in {project_name} by default with pre-configured executors compliant with the FAPI Baseline and FAPI Advanced specifications.
+More details exist in the FAPI section of the link:{adapterguide_link}#_fapi-support[{adapterguide_name}].
 
 [[_client_policy_policy]]
 ===== Policy
 
-A policy consists of several conditions and profiles. The policy can be adopted to clients satisfying any condition of this policy. The policy refers several profiles and all
-executors of these profiles execute their task against the client that this policy is adopted.
+A policy consists of several conditions and profiles. The policy can be adopted to clients satisfying all conditions of this policy. The policy refers several profiles and all
+executors of these profiles execute their task against the client that this policy is adopted to.
 
 
 ==== Configuration
 
-Policies, profiles, conditions, executors can be configured by Admin REST API, which means also an admin console. To do so, there is a tab _Realm_ -> _Realm Settings_ -> _Client Policies_
-, which means an administrator can have client policies per realm.
+Policies, profiles, conditions, executors can be configured by Admin REST API, which means also the Admin Console. To do so, there is a tab _Realm_ -> _Realm Settings_ -> _Client Policies_
+, which means the administrator can have client policies per realm.
 
-The _Global Client Profiles_ are automatically available in each realm. However there are not client policies
-configured by default. This means that administrator is always required to create any client policy if he wants for example the clients of his realm to be FAPI compliant. Global
-profiles cannot be updated, but administrator can easily use them as a template and create his own profile if he wants to do some slight changes in the global profile configurations.
-There is JSON Editor available in the admin console, which simplifies the creation of new profile based on some global profile.
+The _Global Client Profiles_ are automatically available in each realm. However there are no client policies
+configured by default. This means that the administrators are always required to create any client policy if they want for example the clients of his realm to be FAPI compliant. Global
+profiles cannot be updated, but the administrators can easily use them as a template and create their own profile if they want to do some slight changes in the global profile configurations.
```

Comment:
"The administrators" (plural form) – please see comment above.

Reply:
Yes, changing the way you originally proposed as @andymunro confirmed to be fine https://github.com/keycloak/keycloak-documentation/pull/1194#pullrequestreview-675664133